example/examplechecker.py

```diff
@@ -1,128 +1,190 @@
 #!/usr/bin/env python3
 # from src.enochecker import *
-from enochecker import *
+import json
+import secrets
+from typing import Dict
+
+from enochecker import BaseChecker, BrokenServiceException, assert_equals, run
 
 
 class ExampleChecker(BaseChecker):
     """
     Change the methods given here, then simply create the class and .run() it.
-    Magic.
 
     A few convenient methods and helpers are provided in the BaseChecker.
-    ensure_bytes ans ensure_unicode to make sure strings are always equal.
-
-    As well as methods:
-    self.connect() connects to the remote server.
```

Comment:
Will students find these methods? It's probably still good to mention `self.connect()` as there is no example.

Reply:
Maybe a link to https://enowars.github.io/enochecker/enochecker.html#enochecker.enochecker.BaseChecker.connect makes sense?

Reply:
resolved in #53